float64 timestamp
float64 torr